src/components/about/About.test.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import About from "./About";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+
+test("should render component", () => {
+  const history = createMemoryHistory({
+    initialEntries: ["/about"],
+  });
+  render(
+    <Router history={history}>
+      <About />
+    </Router>
+  );
+
+  expect(screen.getByText(/about/i)).toBeInTheDocument();
+  expect(screen.getByText(/app settings/i)).toBeInTheDocument();
+  expect(screen.getByText(/change language/i)).toBeInTheDocument();
+  expect(screen.getByText(/change theme/i)).toBeInTheDocument();
+  expect(screen.getByText(/contact me/i)).toBeInTheDocument();
+  expect(screen.getByText(/source code/i)).toBeInTheDocument();
+  expect(screen.getByText(/report bug/i)).toBeInTheDocument();
```

Comment:
you could consider using [`jest-in-case` and making seperate cases](https://github.com/OpenSourceRaidGuild/atom-pwa/blob/next/src/components/main-menu/MainMenu.test.tsx#L21-L60) for each of these rather than a single test for them all

Reply:
Thanks for this nice suggestion. I have implemented the same.